test/api/transport/transport.transactions.main.js

```diff
@@ -1,257 +0,0 @@
-'use strict';
```

Comment:
@MaciejBaj Do you think it is really necessary? At the end, we are testing the same via http endpoint.